.buildkite/pipelines/periodic-platform-support.yml

```diff
@@ -1,71 +1,46 @@
 steps:
-  - group: platform-support-unix
-    steps:
-      - label: "{{matrix.image}} / platform-support-unix"
-        command: .ci/scripts/run-gradle.sh -Dbwc.checkout.align=true functionalTests
-        timeout_in_minutes: 420
-        matrix:
-          setup:
-            image:
-              - centos-7
-              - debian-10
-              - debian-11
-              - opensuse-leap-15
-              - oraclelinux-7
-              - oraclelinux-8
-              - sles-12
-              - sles-15
-              - ubuntu-1804
-              - ubuntu-2004
-              - ubuntu-2204
-              - rocky-8
-              - rocky-9
-              - rhel-7
-              - rhel-8
-              - rhel-9
-              - almalinux-8
-        agents:
-          provider: gcp
-          image: family/elasticsearch-{{matrix.image}}
-          localSsds: 1
-          localSsdInterface: nvme
-          machineType: custom-32-98304
-        env: {}
-  - group: platform-support-windows
-    steps:
-      - label: "{{matrix.image}} / {{matrix.GRADLE_TASK}} / platform-support-windows"
-        command: |
-          .\.buildkite\scripts\run-script.ps1 bash .buildkite/scripts/windows-run-gradle.sh
-        timeout_in_minutes: 420
-        matrix:
-          setup:
-            image:
-              - windows-2016
-              - windows-2019
-              - windows-2022
-            GRADLE_TASK:
-              - checkPart1
-              - checkPart2
-              - checkPart3
-              - checkRestCompat
-        agents:
-          provider: gcp
-          image: family/elasticsearch-{{matrix.image}}
-          machineType: n1-standard-32
-          diskType: pd-ssd
-          diskSizeGb: 350
-        env:
-          GRADLE_TASK: "{{matrix.GRADLE_TASK}}"
-  - group: platform-support-arm
+  # - group: platform-support-unix
```

Comment:
Presumably this stuff is commented out just to facilitate more efficient testing?

Reply:
This is a draft PR, get out of here

Reply:
Then don't request a review yet 😉 
![image](https://github.com/elastic/elasticsearch/assets/4106672/4e9a6593-bbd6-4885-a9ef-93b1d2a928a9)

Reply:
Did you get a notification or something? I thought it didn't really get pushed out until the PR went to Ready. Maybe that's only with `CODEOWNERS`.

Reply:
I received an email, yes.